app/assets/stylesheets/pages.css.scss

```diff
@@ -144,6 +144,25 @@
   margin-bottom: 10px;
 }
 
+.content .edit_link {
+  font-size: 60%;
+  padding: 0;
+  text-align: right;
+  width: 100%;
+
+  a {
+    border: solid 1px lightgray;
+    border-radius: 3px;
+    color: gray;
```

Comment:
Color `gray` should be written in hexadecimal form as `#808080`<br>Color literals like `gray` should only be used in variable declarations; they should be referred to via variable everywhere else.